ui/e2e/src/fixtures/dashboardTest.ts

```diff
@@ -86,6 +86,18 @@ async function deleteDashboard(projectName: string, dashboardName: string) {
   return result;
 }
 
+// We want to throw on console errors that may mean the app is broken.
+// Some of the libraries we use (e.g. emotion) throw console errors we do not
+// care about at this time. We track those here, so they can be ignored.
+const IGNORE_CONSOLE_ERRORS = [
+  // See https://github.com/emotion-js/emotion/issues/1105
+  'potentially unsafe when doing server-side rendering',
```

Comment:
We should probably fix this at some point, but I don't want to fix it as part of this PR because it's not a trivial fix and we're not focused on supporting SSR right now.